README.md

```diff
@@ -56,6 +56,7 @@ Within each plugin configuration stanza, the "plugin_specific_configuration" obj
 | es_version                                  | string        |  elasticsearch   |   "7.x" | set to "6.x" or "7.x" depending on Elasticsearch version                   |
 | bulk_count                                  | int           |  elasticsearch   |      10 | the number of text chunks processed at once for ES full-text indexing      |
 | read_size                                   | int           |  elasticsearch   | 4194304 | the size of individual text chunks processed for ES full-text indexing     |
+| base64_encoded_tls_credentials              | string        |  elasticsearch   |      "" | the base64 encoded TLS credentials for Elasticsearch                       |
```

Comment:
Consider changing `base64_encoded_tls_credentials` to `authorization_basic_credentials`

Reply:
Done